packages/react-components/src/components/Popover.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import { usePopper } from 'react-popper';
+import * as PopperCore from '@popperjs/core';
+import cx from 'classnames';
+
+type PopoverModifiers = {
+  [key: string]: Partial<PopperCore.StrictModifiers>;
+};
+export interface IPopoverProps {
+  children?: React.ReactNode;
+  className?: string;
+  closeOnEscPress?: boolean;
```

Comment:
I would check with WCAG if we shouldn't always close on ESC. If someone wants to block it, then it could be done by `event.stopPropagation` or something similar.

Reply:
WCAG agrees with you. Reference https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html
👌🏼